src/archive/archive.did

```diff
@@ -124,6 +124,28 @@ type StreamingStrategy = variant {
     };
 };
 
+// management canister types: https://internetcomputer.org/docs/current/references/ic-interface-spec/#ic-candid
+
+type DefiniteCanisterCettings = record {
```

Comment:
Cettings?

also the interface spec uses snake case, should we use that too?

Reply:
I'd rather be consistent with what we have and the maybe change everything to snake case in a later PR.

Reply:
Looks like `snake_case` is the preferred way: https://github.com/dfinity/candid/blob/master/docs/modules/candid-guide/pages/candid-concepts.adoc#reusing-complex-types

Works for me, let's change everything!